Can't build the tidb

install the go and rust using `check_requirement.sh`

failed to build tidb using `build.sh`

build log:
```
building TiDB components in scripts_path
checking if go is installed
go version go1.9.2 linux/amd64
checking if rust is installed
rustc 1.27.0-nightly (565235ee7 2018-05-07)
building TiDB...
Cloning into 'scripts_path/deps/go/src/github.com/pingcap/tidb'...
remote: Counting objects: 1802, done.
remote: Compressing objects: 100% (1548/1548), done.
remote: Total 1802 (delta 235), reused 1020 (delta 158), pack-reused 0
Receiving objects: 100% (1802/1802), 4.57 MiB | 588.00 KiB/s, done.
Resolving deltas: 100% (235/235), done.
Checking connectivity... done.
make parser
make[1]: Entering directory `scripts_path/deps/go/src/github.com/pingcap/tidb'
CGO_ENABLED=0 go build  -o bin/goyacc parser/goyacc/main.go
bin/goyacc -o /dev/null parser/parser.y
Parse table entries: 682338 of 1722972, x 16 bits == 1364676 bytes
bin/goyacc -o parser/parser.go parser/parser.y 2>&1 | egrep "(shift|reduce)/reduce" | awk '{print} END {if (NR > 0) {print "Find conflict in parser.y. Please check y.output for more information."; exit 1;}}'
rm -f y.output
make[1]: Leaving directory `scripts_path/deps/go/src/github.com/pingcap/tidb'
fatal: No names found, cannot describe anything.
CGO_ENABLED=0 go build   -ldflags '-X "github.com/pingcap/tidb/mysql.TiDBReleaseVersion=" -X "github.com/pingcap/tidb/util/printer.TiDBBuildTS=2018-05-08 12:25:25" -X "github.com/pingcap/tidb/util/printer.TiDBGitHash=8d345cd09aedfa72d3018ee5ebdfd452de056e25" -X "github.com/pingcap/tidb/util/printer.TiDBGitBranch=master" -X "github.com/pingcap/tidb/util/printer.GoVersion=go version go1.9.2 linux/amd64"' -o bin/tidb-server tidb-server/main.go
# github.com/pingcap/tidb/statistics
statistics/histogram.go:124:10: undefined: math.Round
make: *** [server] Error 2
```

--- a/scripts/check_requirement.sh
+++ b/scripts/check_requirement.sh
@@ -25,13 +25,13 @@ function install_go {
     echo "Intall go ..."
     case "$OSTYPE" in 
         linux*) 
-            curl -L https://storage.googleapis.com/golang/go1.9.2.linux-amd64.tar.gz -o golang.tar.gz
+            curl -L https://storage.googleapis.com/golang/go1.10.2.linux-amd64.tar.gz -o golang.tar.gz
             ${SUDO} tar -C /usr/local -xzf golang.tar.gz
             rm golang.tar.gz
         ;;
 
         darwin*)
-            curl -L https://storage.googleapis.com/golang/go1.9.2.darwin-amd64.tar.gz -o golang.tar.gz
+            curl -L https://storage.googleapis.com/golang/go1.10.2.darwin-amd64.tar.gz -o golang.tar.gz
             ${SUDO} tar -C /usr/local -xzf golang.tar.gz
             rm golang.tar.gz
         ;;
@@ -94,8 +94,8 @@ if which go &>/dev/null; then
     # requires go >= 1.8
     GO_VER_1=`go version | awk 'match($0, /([0-9])+(\.[0-9])+/) { ver = substr($0, RSTART, RLENGTH); split(ver, n, "."); print n[1];}'`
     GO_VER_2=`go version | awk 'match($0, /([0-9])+(\.[0-9])+/) { ver = substr($0, RSTART, RLENGTH); split(ver, n, "."); print n[2];}'`
-    if [[ (($GO_VER_1 -eq 1 && $GO_VER_2 -lt 8)) || (($GO_VER_1 -lt 1)) ]]; then
-        echo "Please upgrade Go to 1.8 or later."
+    if [[ (($GO_VER_1 -eq 1 && $GO_VER_2 -lt 10)) || (($GO_VER_1 -lt 1)) ]]; then
+        echo "Please upgrade Go to 1.10 or later."
         exit 1
     fi
 else

